setup.py

```diff
@@ -13,37 +13,15 @@
 with open(os.path.join(here, 'qtawesome', '_version.py')) as f:
     exec(f.read(), {}, version_ns)
 
-LONG_DESCRIPTION = """
-.. image:: https://img.shields.io/pypi/v/QtAwesome.svg
-   :target: https://pypi.python.org/pypi/QtAwesome/
-   :alt: Latest PyPI version
-
-.. image:: https://img.shields.io/pypi/dm/QtAwesome.svg
-   :target: https://pypi.python.org/pypi/QtAwesome/
-   :alt: Number of PyPI downloads
-
-QtAwesome - Iconic Fonts in PyQt and PySide applications
-========================================================
-
-QtAwesome enables iconic fonts such as Font Awesome and Elusive Icons in PyQt and PySide applications.
-
-It is a port to Python - PyQt / PySide of the QtAwesome C++ library by Rick Blommers.
-
-.. code-block:: python
-
-    # Get icons by name.
-    fa_icon = qta.icon('fa5s.flag')
-    fa_button = QtGui.QPushButton(fa_icon, 'Font Awesome!')
-
-    asl_icon = qta.icon('ei.asl')
-    elusive_button = QtGui.QPushButton(asl_icon, 'Elusive Icons!')
-"""
+with open(os.path.join(here, 'README.md'), encoding='utf-8') as f:
```

Comment:
`encoding` is not a valid keyword for `open` in Python 2. I think you can use `io.open` instead.